packages/scaffold/src/views/w3m-update-email-wallet-waiting-view/index.ts

```diff
@@ -0,0 +1,105 @@
+import { customElement } from '@web3modal/ui'
+import { LitElement, html } from 'lit'
+import styles from './styles.js'
+import { RouterController, ConnectorController, SnackController } from '@web3modal/core'
+import { state } from 'lit/decorators.js'
+
+@customElement('w3m-update-email-wallet-waiting-view')
+export class W3mUpdateEmailWalletWaitingView extends LitElement {
+  public static override styles = styles
+
+  // -- Members ------------------------------------------- //
+  protected readonly email = RouterController.state.data?.email
+
+  protected readonly emailConnector = ConnectorController.getEmailConnector()
+
+  public constructor() {
+    super()
+    this.listenForEmailUpdateApproval()
+  }
+
+  // -- State & Properties -------------------------------- //
+  @state() private loading = false
+
+  // -- Render -------------------------------------------- //
+  public override render() {
+    if (!this.email) {
+      throw new Error('w3m-update-email-wallet-waiting-view: No email provided')
+    }
+    if (!this.emailConnector) {
+      throw new Error('w3m-update-email-wallet-waiting-view: No email provided')
```

Comment:
```suggestion
      throw new Error('w3m-update-email-wallet-waiting-view: No email connector provided')
```

Having a more accurate description would help us while we debug issues